tests/Unit/PointwiseFunctions/GeneralRelativity/Test_WeylElectric.cpp

```diff
@@ -35,5 +46,5 @@ SPECTRE_TEST_CASE("Unit.PointwiseFunctions.GeneralRelativity.WeylElectric",
 
   GENERATE_UNINITIALIZED_DOUBLE_AND_DATAVECTOR;
 
-  CHECK_FOR_DOUBLES_AND_DATAVECTORS(test_weyl_electric, (1, 2, 3));
+  CHECK_FOR_DOUBLES_AND_DATAVECTORS(test_weyl_electric_scalar, (1, 2, 3));
 }
```

Comment:
test the compute tag

Reply:
ping on this, you need to add WeylElectricScalar to the compute tags in `test_compute_item_in_databox`